americangut/util.py

```diff
@@ -492,7 +492,7 @@ def clean_and_reformat_mapping(in_fp, out_fp, body_site_column_name,
             body_site = body_site.split('_', 1)[-1].replace("_", " ")
         elif body_site.startswith('UBERON:'):
             body_site = body_site.split(':', 1)[-1]
-        elif body_site in ['NA', 'unknown', '', 'no_data', 'None']:
+        elif body_site in ['NA', 'unknown', '', 'no_data', 'None', 'Unknown']:
```

Comment:
I think case does matter. AG round 1-22 processing failed in notebook 4 (metadata); when I added Unknown, uppercase, to my local file, the notebook ran successfully.